cityofaustin/atd-vz-data
# 12516 add user email export button

## Associated issues

Closes [#12516](https://github.com/cityofaustin/atd-data-tech/issues/12516) which allows one to download all the emails in the VZE Users tab.

## Testing
**URL to test:** 
https://deploy-preview-1294--atd-vze-staging.netlify.app/#/users

**Steps to test:**
- go to the users page and click the 'copy user emails' button at the top right of the table
- you should see a loading spinner while fetching the user list from the the auth0 api
- this may take a few seconds because we need to make multiple asynchronous api requests
- once that is done, the button should reappear and a success popover should display for two seconds and then disappear
- open a text doc and paste the user list from your clipboard!
- note: if you click the button again the previously fetched data will be copied to your clipboard. since fetching the data is somewhat resource intensive, it will only refetch the data if the number of users somehow changes

---
#### Ship list
